webapp/i18n/en.json

```diff
@@ -246,45 +254,25 @@
   "ViewTitle.untitled-board": "Untitled board",
   "WelcomePage.Description": "Boards is a project management tool that helps define, organize, track and manage work across teams, using a familiar kanban board view",
   "WelcomePage.Explore.Button": "Take a tour",
-  "WelcomePage.NoThanks.Text": "No thanks, I'll figure it out myself",
   "WelcomePage.Heading": "Welcome To Boards",
+  "WelcomePage.NoThanks.Text": "No thanks, I'll figure it out myself",
   "Workspace.editing-board-template": "You're editing a board template.",
   "calendar.month": "Month",
   "calendar.today": "TODAY",
   "calendar.week": "Week",
+  "cloudMessage.learn-more": "Learn more",
   "default-properties.badges": "Comments and Description",
   "default-properties.title": "Title",
-  "error.relogin": "Log in again",
+  "error.page.title": "Sorry, something went wrong",
+  "generic.previous": "Previous",
   "login.log-in-button": "Log in",
   "login.log-in-title": "Log in",
-  "error.workspace-undefined": "Not a valid workspace.",
-  "error.page.title": "Sorry, something went wrong",
-  "error.not-logged-in": "Your session may have expired or you're not logged in.",
-  "error.back-to-home": "Back to Home",
-  "error.back-to-boards": "Back to boards",
-  "error.go-login": "Log in",
-  "error.unknown": "An error occurred.",
   "login.register-button": "or create an account if you don't have one",
   "register.login-button": "or log in if you already have an account",
-  "OnboardingTour.OpenACard.Title": "Open a card",
-  "OnboardingTour.OpenACard.Body": "Open a card to explore the powerful ways that Boards can help you organize your work.",
-  "OnboardingTour.AddProperties.Title": "Add properties",
-  "OnboardingTour.AddProperties.Body": "Add various properties to cards to make them more powerful!",
-  "OnboardingTour.AddComments.Title": "Add comments",
-  "OnboardingTour.AddComments.Body": "You can comment on issues, and even @mention your fellow Mattermost users to get their attention.",
-  "OnboardingTour.AddDescription.Title": "Add description",
-  "OnboardingTour.AddDescription.Body": "Add a description to your card so your teammates know what the card is about.",
-  "OnboardingTour.AddView.Title": "Add a new view",
```

Comment:
I'm not sure why these got removed. I don't think any of these should have been removed.
https://github.com/mattermost/focalboard/blob/release-7.0/webapp/src/components/onboardingTour/addView/add_view.tsx#L19

Reply:
`i18n-extract` wasn't pattern matching files deep enough—fixed with https://github.com/mattermost/focalboard/pull/3071/commits/941994f829396a215065576cb85b018705befb5d